rqd/tests/rqconstants_tests.py

```diff
@@ -128,7 +128,7 @@ def test_facility(self):
         self.assertEqual(rqd.rqconstants.DEFAULT_FACILITY, "test_facility")
 
         machine = self.makeRqMachine()
-        self.assertEqual(machine.renderHost.facility, "test_facility")
```

Comment:
This should be `test_facility` because of `MockConfig` override with `DEFAULT_FACILITY =  test_facility`.

Reply:
I see, it was failing in our test environment, so it's probably an issue on our end that needs to be resolved